app/templates/admin/admin.html

```diff
@@ -35,9 +35,18 @@
 {% block body %}
 {% set active_page = active_page|default('Home') -%}
 <nav class="nav">
+    <div class="ham">
```

Comment:
Let's rename this class (everywhere!) to "toggle" because it contains both the ham (three bars) and the close (an x)